File: platform/sdk/python/examples/cloud/cloud-017_iam_get_policy.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.iamPutPolicy("ReadOnly", {"Statement": []})
result = client.iamGetPolicy("ReadOnly")
if not result["found"] or not result["policy"]:
    sys.exit(1)
print("XJ-OK: cloud-017_iam_get_policy")
sys.exit(0)

File: platform/sdk/nodejs/examples/cloud/cloud-017_iam_get_policy.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.iamPutPolicy("ReadOnly", { Statement: [] });
const result = client.iamGetPolicy("ReadOnly");
if (!result.found || !result.policy) process.exit(1);
console.log("XJ-OK: cloud-017_iam_get_policy");
process.exit(0);
